src/reporting/tests/test_pvmon/test_models.py

```diff
@@ -0,0 +1,162 @@
+import pytest
+from django.test import TestCase
+
+from pvmon.models import PVName
+from pvmon.models import PV
+from pvmon.models import PVCache
+from pvmon.models import PVString
+from pvmon.models import PVStringCache
+from pvmon.models import MonitoredVariable
+from report.models import Instrument
+
+
+class PVNameTest(TestCase):
+    def test_str(self):
+        pvname = PVName(name="test")
+        self.assertEqual(str(pvname), "test")
+
+    def test_max_length(self):
+        pvname = PVName(name="test")
+        max_len = pvname._meta.get_field("name").max_length
+        self.assertEqual(max_len, 50)
+
+
+class PVTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
Do the models not need a `tearDownClass` to delete these?

Reply:
opps, I forgot to do the cleanup again.  Adding it now